pkg/testcase/selinux.go

```diff
@@ -0,0 +1,120 @@
+package testcase
+
+import (
+	"fmt"
+	"log"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/rancher/distros-test-framework/factory"
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/shared"
+)
+
+// TestSelinuxEnabled Validates that containerd is running with selinux enabled in the config
+func TestSelinuxEnabled() {
+	product, err := shared.GetProduct()
+	if err != nil {
+		log.Println(err)
+	}
+
+	ips := shared.FetchNodeExternalIP()
+	selinuxConfigAssert := "selinux: true"
+	selinuxContainerdAssert := "enable_selinux = true"
+
+	for _, ip := range ips {
+		err := assert.CheckComponentCmdNode("cat /etc/rancher/"+product+"/config.yaml", ip, selinuxConfigAssert)
+		Expect(err).NotTo(HaveOccurred())
+		errCont := assert.CheckComponentCmdNode("sudo cat /var/lib/rancher/"+product+"/agent/etc/containerd/config.toml", ip, selinuxContainerdAssert)
+		Expect(errCont).NotTo(HaveOccurred())
+	}
+}
+
+// TestSelinuxVersions Validates container-selinux version, rke2-selinux version and rke2-selinux version
+func TestSelinuxVersions() {
+	product, err := shared.GetProduct()
+	if err != nil {
+		log.Println(err)
+	}
+	cluster := factory.AddCluster(GinkgoT())
+
+	serverCmd := "rpm -qa container-selinux " + product + "-server " + product + "-selinux"
```

Comment:
here to line 66 its to confusing.....and a lot of repetition unnecessary 

use some data structure like map or switch to handle this

Reply:
What do you think now? is better?

```
	serverCmd := "rpm -qa container-selinux rke2-server rke2-selinux"
	if product == "k3s" {
		serverCmd = "rpm -qa container-selinux k3s-selinux"
	}

	serverAsserts := []string{"container-selinux", "rke2-selinux", "rke2-server"}
	if product == "k3s" {
		serverAsserts = []string{"container-selinux", "k3s-selinux"}
	}
```